src/main/java/edu/ucla/library/bucketeer/verticles/FinalizeJobVerticle.java

```diff
@@ -292,6 +290,7 @@ private void sendSlackMessage(final String aChannelId, final String aMessageText
      */
     private void sendSlackMessage(final String aChannelId, final String aMessageText, final Job aJob,
```

Comment:
Is it true that the only changes to FinalizeJobVerticle's logic were made to sendSlackMessage? (And the rest of the diff is code formatting?)

Reply:
Yes